### DISCUSSION TITLE
How can i configure RKE2 to use a private container registries alongside with default docker hub?

### DISCUSSION THREAD

**Discussion by apriss** [rancher/rke2#3986] [Q&A] [answered]
Hi,

Is it possible to use a private container registry but the other hand still enable the default container registry (docker Hub)? If it's possible, so how can I configure this in /etc/rancher/rke2/regitries.yaml?

Regards

**Comment by brandond** [comment, marked as answer]
Do you want to use your private registry as a mirror for docker hub, or do you just want to explicitly pull some of your images from a private registry?

**Reply by apriss** [reply]
I want to pull some images from a private registry but still be able to pull images from docker hub

Because if I try to use the mirror registry configuration as in the following [RKE2 documentation](https://docs.rke2.io/install/containerd_registry_configuration) I can no longer pull the image from docker hub

**Reply by brandond** [reply]
Just pull images from it, by referring to images on your registry in the pod spec. You don't need to add anything to registries.yaml just to pull images. That's only needed if you want to configure mirroring, tls, or credentials.
